### PR TITLE
Add codecept config for functional and crossbrowser tests

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/CIV-844

### CODE REVIEW

**Review by LeonardoPalmeiro — APPROVED** [review]
all good

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -1,8 +1,11 @@
 #!groovy
 
 properties([
-// H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
-pipelineTriggers([cron('H 05 * * *')])
+    // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
+    pipelineTriggers([cron('H 05 * * *')]),
+    parameters([
+        string(name: 'TEST_URL', defaultValue: 'https://civil-citizen-ui-pr-97.service.core-compute-preview.internal', description: 'The URL you want to run tests against'),
```
Comment (mounikahmcts) [inline Jenkinsfile_nightly:7]: Will point to AAT later. This is for testing

File: src/test/config.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { PropertiesVolume } from '../main/modules/properties-volume';
+import { Application } from 'express';
+
+if (!process.env.TEST_PASSWORD) {
+  new PropertiesVolume().enableFor({ locals: { developmentMode: true } } as unknown as Application);
+}
+
+export const config = {
```
Comment (commitBlob) [inline src/test/config.ts:8]: looking at the attributes there are different naming conventions used (TEST_URL = snake_case, TestHeadlessBrowser = PascalCase, features = lowercase).
 For consistency I would advice to only use one.

**Review by commitBlob — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -17,7 +17,10 @@
     "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
     "sonar-scan": "sonar-scanner",
     "test:smoke": "jest -c jest.smoketest.config.js",
-    "test:functional": "jest -c jest.functionaltest.config.js",
+    "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs --config ./src/test/functional/codecept.conf.js run-workers 2 parallel",
```
[the remark this answers is not in the document]
Comment (mounikahmcts) [reply package.json:20]: Updated readme

**Review by RosDiaf — APPROVED** [review]

**Pull request merged**